app/src/main/res/xml/network_security_config.xml

```diff
@@ -1,4 +1,7 @@
 <network-security-config>
+    <domain-config cleartextTrafficPermitted="true">
+        <domain includeSubdomains="true">token-price-service.radixdlt.com</domain>
+    </domain-config>
```

Comment:
@GhenadieVP 
I am raising this again, it was necessary to add this here otherwise it doesn't work.

The `cleartextTrafficPermitted` flag is one of the options in Android’s [Network Security Configuration](https://developer.android.com/privacy-and-security/security-config#CleartextTrafficPermitted) file. 
From Android 9 (API level 28) and higher, **this is set by default to false and it is intended to prevent insecure communication attempts using clear-text HTTP originating from Android applications.**